os/board/common/partitions.c

```diff
@@ -24,18 +24,18 @@
 #include <debug.h>
 #include <errno.h>
 #include <string.h>
-#include <sys/types.h>
 #include <stdlib.h>
 #include <stdio.h>
+#include <sys/types.h>
```

Comment:
It is better to keep original or move up after <tinyara/config.h>. No need to move down.

Reply:
modify